notebooks/03_raster_zonal_stats.ipynb

```diff
@@ -354,8 +354,8 @@
      "name": "stdout",
```

Comment:
<p>Suggestion: I'm thinking that instead of adding an example of an unsupported option, maybe we can leave a callout note instead that warns users that <code>gdal</code> output is unsupported and <code>pandas</code> is the default.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/geowrangler/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>